index.html

```diff
@@ -161,32 +169,37 @@ <h2 class="title cards">Multi-Post Stories</h2>
             </li>
         </ul>
 
-        <a href="#" class="button project">See Project</a>
+        <a href="#" class="button project" id="last">See project</a>
     </div>
+</div>
     </section>
 
     <section id="myself">
 
-        <h2 class="title">
+        <div>
+            <h2 class="title" id="last_h2">
```

Comment:
As suggested by the previous reviewer, Please increase the `font-size` for the `About myself` to `40px` as used in the Figma design instead of `30px` to allow it to be visible enough